22_pipeslab/pipeslab.txt

```diff
@@ -2,20 +2,47 @@ For each of the following, type a single command line that performs:
 
   1. Print the number of lines from "ls -l" which represent a path that has execute permission to their owner.
   Answer: 
+I used ls -la to get all the content (including hidden)
+It works for me in such way:
+
+$ ls -la | cut -c 4 | grep x | cat -n | tail -1 | cut -f1
```

Comment:
Nice! I love the use of `cat -n` to count, however an easier way is to add `-c` to grep i.e. `ls -la | cut -c 4 | grep -c x`